### PR TITLE
[Web-38] color palette for course menu

### PR DESCRIPTION
**Pull request by anthopark**
https://user-images.githubusercontent.com/42628074/217159095-c3e51dd0-e3a3-446b-b883-8284bb331830.mov

* Implement color palette component in the course menu
* update course color upon color selection in the color palette.

### CODE REVIEW

**Comment by anthopark** [comment]
[[Web] Add Color palette for SortableCourseItem menu](https://trello.com/c/dxHmkLKf/38-web-add-color-palette-for-sortablecourseitem-menu)

File: new-nextjs-client/src/entities/app-user.ts [hunk 1]
```diff
@@ -16,7 +26,7 @@ class AppUser implements IAppUser {
   private _years: number[] = [];
   private _authToken: string | null = null;
   private _degreePlan = new DegreePlan();
-  private _courseBag: Course[] = [
+  private _courseBag: ICourse[] = [
```
Comment (emilyPhee) [inline new-nextjs-client/src/entities/app-user.ts:29]: Could you let me know what the `ICourse` array name stands for here?

**Review by emilyPhee — APPROVED** [review]
Awesome work! Thank you for working on the course menu. 
I just left a question, thanks!

File: new-nextjs-client/src/entities/app-user.ts (hunk 1 above)
Comment (anthopark) [reply new-nextjs-client/src/entities/app-user.ts:29]: Interface!

**Pull request merged**